### ISSUE TITLE
MASS MERGE : Improve behaviour when same recording on several tracks

### ISSUE THREAD

**Issue by jesus2099** [jesus2099/konami-command#53]
It’s a bug from the start.
First thought for local release is that I should stop using `recid2trackIndex` and always use some `querySelectorAll` to update info and to clean tracks.
I have no first thought for remote release, for which I don’t feel a problem yet.

**Comment by jesus2099** [comment]
Example of a release that has tracks with same recording : https://musicbrainz.org/release/8fc541fb-7601-4a02-b859-b1d9789ddde7

**Comment by jesus2099** [comment]
At the same time, remove the « remote release must be different than local » limitation.

**Comment by pankkake** [comment]
I agree, there are legitimate cases for merging a release to itself!

**Comment by jesus2099** [comment]
For example on https://musicbrainz.org/release/d12007e9-e280-49f4-ab16-a045b88131bb
everything merged OK but all buttons remained at 2/2 and inline merge form were not cleaned up.
